src/stylesheets/components/_components.share.scss

```diff
@@ -1,10 +1,31 @@
 .c-share {
   display: flex;
+  position: relative;
+  padding-bottom: 25px;
+  @include mq ($from: tablet) {
+    padding-bottom: 32px;
+  }
   @include inuit-font-size($font-size-p,$line-height-p);
+  &::after {
+    content: "";
+    display: block;
+    position: absolute;
+    height: 1px;
+    background: $alto;
```

Comment:
update to `background-color` - the `background` selector is quite broad and hard to tell what we are doing if left alone 👍